osu.Game.Rulesets.Osu/Difficulty/OsuPerformanceCalculator.cs

```diff
@@ -98,11 +98,11 @@ private double computeAimValue()
 
             double approachRateFactor = 0.0;
             if (Attributes.ApproachRate > 10.33)
-                approachRateFactor += 0.4 * (Attributes.ApproachRate - 10.33);
+                approachRateFactor = Attributes.ApproachRate - 10.33;
             else if (Attributes.ApproachRate < 8.0)
-                approachRateFactor += 0.01 * (8.0 - Attributes.ApproachRate);
+                approachRateFactor = 0.025 * (8.0 - Attributes.ApproachRate);
 
-            aimValue *= 1.0 + Math.Min(approachRateFactor, approachRateFactor * (totalHits / 1000.0));
+            aimValue *= 1.0 + (0.03 + 0.37 * (1.0 / Math.Exp(-(0.0007 * (totalHits - 400))))) * approachRateFactor;
```

Comment:
This formula differs from the one given in the desmos link in no less than two places.

```
this PR: 1.0 + (0.03 + 0.37 * (1.0 / Math.Exp(-(0.0007 * (totalHits - 400))))) * approachRateFactor
desmos:  1.0 + (0.03 + 0.37 * (1.0 / (1.0 + Math.Exp(-(0.007 * (totalHits - 400)))))) * approachRateFactor
```

i.e. one of the exponential factors is off by a factor of 10, and the entire denominator of the expression with the exponential is missing a `1 +`. Those mistakes completely change the characteristics of the curve; see [revised desmos](https://www.desmos.com/calculator/hkcnmarvtp?lang=pl). Red curve is proposed in the pull description, orange is the actual one in code right now.

I'm not sure how much this change was tested (if at all) but it might need more of that.

Reply:
Oh wow, guess my eyes weren't working at all when I PR'd this. It is indeed very incorrect, what's in the desmos should be considered the proper version. I'll fix it in a sec and add a few profiles to make sure it's looking correct.

There was indeed no code testing since I assumed it to be small enough change. Having pp changes visible wouldn't hurt though but I'm not planning to set up leaderboards for this change so someone else might have to do it.